src/_guides/libraries/useful-libraries.md

```diff
@@ -24,141 +32,55 @@ in alphabetical order:
-for access to the Firebase JavaScript API, and the
-[dart:html library]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-html/dart-html-library.html)
-for low-level HTML programming.
-
-**Learn more:** [Dart Webdev]({{site.webdev}})
-
-### Server-side libraries
-
-If you write servers or command-line applications, check out
-[dart:io]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/dart-io-library.html)
-and related libraries.
-
-**Learn more:** [server-side Dart]({{site.dart_vm}})
-
-### Flutter libraries
 
-If you write mobile apps, check out Flutter.
-The core libraries distributed with the Flutter SDK are documented at
-[docs.flutter.]({{site.flutter_api}}) To import these libraries,
-follow the instructions in [Importing libraries from
-packages](/tools/pub/get-started#importing-libraries-from-packages).
 
-**Learn more:** [Flutter API]({{site.flutter_api}})
+## Specialized packages
 
-## Resources
+Here are some tips for finding packages that are more specialized,
+such as packages for mobile (Flutter) and web development.
 
-Use the following resources to learn more about libraries and library packages.
+### Flutter packages
 
-### Importing and using libraries
+See [Using packages]({{site.flutter}}/docs/development/packages-and-plugins/using-packages)
+on the Flutter site.
+Or search for [Flutter packages.]({{site.pub}}/flutter)
 
-* [Using libraries](/guides/language/language-tour#libraries-and-visibility),
-  a section in the [language tour](/guides/language/language-tour)
-* [Importing library
-  files](/guides/libraries/create-library-packages#importing-library-files),
-  a section in [Create Library Packages](/guides/libraries/create-library-packages)
-* [Pub docs](/tools/pub), particularly
-  [Pub Package Layout Conventions](/tools/pub/package-layout) and
-  [Dependency sources](/tools/pub/dependencies#dependency-sources)
 
-### Creating library packages
+### Web packages
 
-* [Create Library Packages](/guides/libraries/create-library-packages)
+See [Web libraries and packages](/web/libraries).
+Or search for [web packages.]({{site.pub}}/web)
 
-### Using specific libraries and packages
 
-* [A Tour of the Dart Libraries](/guides/libraries/library-tour), which
-  gives examples of many commonly used dart:* APIs.
-* [Unboxing Packages]({{site.news}}/search/label/Unboxing%20Packages)
-  posts, written by written by Natalie Weizenbaum and published on the
-  [Dart news site.]({{site.news}})
-  This page links to some of Natalie's posts, but she covers other packages
-  not mentioned here, such as stream_channel, vm_service_client, and json_rpc_2.
+### Command-line and server packages
 
-### Packages contributed by the community
+The [Pub site]({{site.pub}}) doesn't currently support
+limiting your search to packages that support these kinds of apps.
+You can, however, search for words that describe the functionality you need.
 
-* [Pub site]({{site.pub}})
+Command-line apps often use the following packages:
```

Comment:
A few more ideas:

* archive
* intl
* yaml
* completion

Reply:
I don't want to double-cover packages, so I might mention the ones that are already in a table above (intl, yaml) but not include them in this table. What do you think?

Reply:
sgtm